docs/source/supported/rivanna.rst

```diff
@@ -0,0 +1,112 @@
+====================
+Rivanna (UVA)
+====================
+
+Platform user guide
+===================
+
+https://www.rc.virginia.edu/userinfo/user-guide/
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``uva.rivanna*`` - ``MPIRUN``
+
+* Configuration per node (per platform ID)
+
+  * ``uva.standard`` (1 node)
+
+    * 40 CPU cores (1 thread per core)
+    * 375GB GiB of memory
+

```

Comment:
same as for Bridges (and this also contains wrong example as platform ID mixed with queue)